README.md

```diff
@@ -2,8 +2,8 @@ RISC-V Debug Specification
 ==========================
 
 You may be looking for one of the following pre-built PDFs:
-* [Latest stable](https://github.com/riscv/riscv-debug-spec/blob/master/riscv-debug-stable.pdf)
-* [Latest release](https://github.com/riscv/riscv-debug-spec/blob/release/riscv-debug-release.pdf)
+* [Latest stable](https://github.com/riscv/riscv-debug-spec/releases/latest/riscv-debug-stable.pdf)
+* [Latest release](https://github.com/riscv/riscv-debug-spec/releases/latest/riscv-debug-release.pdf)
```

Comment:
These links don't go anywhere for me.

Reply:
Not yet.  I believe that they will once CI runs.  For instance, the equivalent link works on various other RISC-V specs.

Reply:
I'll file another for this.